actions/install/action.yml

```diff
@@ -9,6 +9,10 @@ inputs:
     description: The version of the Catalyst-CI CLI to install
     required: false
     default: latest
+  local:
+    description: Use the local Catalyst-CI CLI
```

Comment:
We should indicate in the description that this is _building_ a local version of the CLI. Otherwise, it might be confusing about where it's coming from.